packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.spec.ts

```diff
@@ -157,6 +163,45 @@ describe('Design Token Renderer', () => {
     });
   });
 
+  describe('getTokenSorterFromRegExpList', () => {
+    it('should sort properly based on regExps', () => {
+      const regExps = [
+        /override$/,
+        /shadow/
+      ];
+      const list = Array.from(designTokens.values());
+      const sortedTokens = getTokenSorterFromRegExpList(regExps)(designTokens)(list);
+
+      expect(list.findIndex(({ tokenReferenceName }) => tokenReferenceName === 'example.var-expect-override'))
+        .toBeGreaterThan(list.findIndex(({ tokenReferenceName }) => tokenReferenceName === 'example.var1'));
```

Comment:
We are doing several time `list.findIndex(({ tokenReferenceName }) => tokenReferenceName === 'example.var1')` for example
we can grouped them in a variable to improve a bit the performances no ?